numpy/f2py/crackfortran.py

```diff
@@ -1480,7 +1480,7 @@ def analyzeline(m, case, line):
                             else:
                                 expanded_list.append(match.strip())
                         matches = expanded_list
-                    new_val = "(/{}/)".format(", ".join(matches)) if vdim else matches[idx]
+                    new_val = f'(/{", ".join(matches)}/)' if vdim else matches[idx]
```

Comment:
This and elsewhere in this file would be easier to read in this style if you bind `", ".join(matches)` to a variable name and use that here, I had to squint for a while trying to parse the nested quotes.

Reply:
Good suggestion, would be happy to make those changes.  Since it is sometimes unclear what the code is doing at a certain section of the program, would it be alright if I use a "catch-all" variable name (perhaps `joined_elements`) to store these values?

In the cases where it is more clear what's being joined, I can name the variable something more specific to the operation that's occurring (like here, I would name the variable something along the lines of `all_matches` or `joined_matches`).  Or would you prefer that I use the same variable name (`joined_elements`) for all of these kinds of operations?

Reply:
Not a big deal, whatever you think is most understandable.

Reply:
Okay, will do.